src/module/entities/helper.js

```diff
@@ -1323,6 +1323,10 @@ module.exports = class UserProjectsHelper {
 	static update(entityId, bodyData) {
 		return new Promise(async (resolve, reject) => {
 			try {
+			    // Check if metaInformation is provided and contains externalId
```

Comment:
@MallanagoudaB you should do this in validation only . 
refer the fetch function validation from this file it may help :https://github.com/ELEVATE-Project/samiksha-service/blob/9edd8d28a834d4febbb5d9968a86098819cd78e7/module/reports/validator/v1.js